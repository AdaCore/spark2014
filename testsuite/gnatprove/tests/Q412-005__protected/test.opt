ALL XFAIL GNATprove crashes on protected function called to define constant
