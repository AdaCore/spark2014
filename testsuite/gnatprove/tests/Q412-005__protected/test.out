pack.adb:10:25: warning: statement has no effect
pack.ads:4:49: info: postcondition proved (CVC4: 1 VC; Trivial: 1 VC)
